stories/assets/js/Tab_custom.js

```diff
@@ -0,0 +1,11 @@
+/* Tab JS start custom */
+export function TabFun() {
+	$('#tab.tab__items .tab__item').click(function(){
```

Comment:
@harishkumar-srijan i think we can give '#tab .tab__item'

Reply:
Done